torch/_functorch/_aot_autograd/runtime_wrappers.py

```diff
@@ -1991,17 +1993,50 @@ def _backward_impl(ctx, all_args):
                     context = torch._C._DisableAutocast if disable_amp else nullcontext
                     with tracing(saved_context), compile_context(
                         saved_compile_context
-                    ), context(), track_graph_compiling(aot_config, "backward"):
-                        CompiledFunction.compiled_bw = aot_config.bw_compiler(
-                            bw_module, placeholder_list
-                        )
-                        # Maybe save cache entry
-                        if try_save_cache_entry is not None:
-                            try_save_cache_entry(
-                                CompiledFunction.compiled_bw,
-                                fw_metadata,
-                                aot_config,
+                    ), context(), track_graph_compiling(
+                        aot_config, "backward"
+                    ), METRICS_CONTEXT, dynamo_timed(
+                        # TODO: Capture this under "entire_frame_compile" or a new name?
+                        # It would seem to serve the same purpose when we capture the full
+                        # compile in convert_frame_py, but "frame" may not make sense here.
+                        "entire_frame_compile",
+                        fn_name="backward._backward_impl",
+                        log_pt2_compile_event=False,
+                        # TODO: How do we want to record this one in dynamo_compile?
+                        dynamo_compile_column="dynamo_cumulative_compile_time_us",
+                    ):
```

Comment:
The changes below are analogous to the code in convert_frame: gather up the settings we want to ultimately appear in the CompilationMetrics (not otherwise gathered via dynamo_timed) and put them in the MetricsContext.